example/src/BUILD.bazel

```diff
@@ -17,6 +17,12 @@ filegroup(
     tags = ["markdown"],
 )
 
+filegroup(
+    name = "tf",
+    srcs = ["hello.tf"],
+    tags = ["terraform"],
```

Comment:
nit: add a comment about the load-bearing function of this tag, as this is an example it should be instructive

Reply:
It's missing from the markdown target above.